[AIRFLOW-6392] Remove cyclic dependency baseoperator <-> helpers (#6950)
mistercrunch/Airflow: UPDATING.md

-->

### Chain and cross_downstream moved from helpers to BaseOperator

> The chain and cross_downstream methods are now moved to airflow.models.baseoperator module from

Reviewer: ```suggestion
The `chain` and `cross_downstream` methods are now moved to `airflow.models.baseoperator` module from
```

Author: It's already solved as well :)

> airflow.utils.helpers module where they belong.

Reviewer: ```suggestion
`airflow.utils.helpers` module where they belong.
```

Author: Yep. I will also update the description slightly.

Helpers module is supposed to contain standalone helper methods that can be imported by all classes.
The `chain` method and `cross_downstream` method both use BaseOperator and if any other package imports

> any of the helpers classes it automatically creates an implicit dependency to BaseOperator which

Reviewer: ```suggestion
any of the helpers classes or functions it automatically creates an implicit dependency to BaseOperator which
```

Author: Yep. I also reworded it slightly.

can often lead to cyclic dependencies.

More infomration in [AIFLOW-6392](https://issues.apache.org/jira/browse/AIRFLOW-6392)

> In Airflow 1.10 you imported those two methods like this:

Reviewer: ```suggestion
In Airflow <2.0 you imported those two methods like this:
```

Author: 👍
